.travis.yml

```diff
@@ -61,12 +61,12 @@ compiler:
 
 env:
   # With or without cross-compiling:
-  - DYNAMORIO_CROSS_ONLY=yes
+  - DYNAMORIO_CROSS_AARCHXX_LINUX_ONLY=yes
```

Comment:
Android builds are for ARM too, so I thought it would be a good idea to have Linux in there, to differentiate.